queries/vm_backup.kusto

```diff
@@ -0,0 +1,24 @@
+let VMs = MonitoringResources_CL 
+| where TimeGenerated > ago(13h)
+| where type_s == "microsoft.compute/virtualmachines"
+| summarize arg_max(TimeGenerated, *) by name_s
+| project vmResourceID = ResourceId, name = tolower(name_s), managedby = tags_managedby_s, osType = properties_storageProfile_osDisk_osType_s;
+AddonAzureBackupJobs
+| where JobStatus == "Failed"
+| where JobOperation =="Backup"
+| extend vm = extract("(iaasvmcontainerv2;.+?;)(.+)", 2, BackupItemUniqueId)
+| extend vault = extract("(microsoft.recoveryservices/vaults/)(.+)$", 2, tostring(_ResourceId))
+| project TimeGenerated, JobFailureCode, vm, vault, _ResourceId, JobOperationSubType, JobStartDateTime, BackupItemUniqueId, AdHocOrScheduledJob
+| join kind = leftouter(VMs) on $left.vm == $right.name
+| where managedby == "q.beyond"
```

Comment:
Do we normalize the tags in our table? Otherwise we may want to check case insensitive.
```suggestion
| where tolower(managedby) == "q.beyond"
```

Reply:
I've not seen anything else for this tag yet, but doesn't hurt to do it. I've added a case insensitive =~ match.